pkgs/applications/editors/emacs/generic.nix

```
@@ -45,6 +44,8 @@
   else if withMotif then "motif"
   else if withAthena then "athena"
   else "lucid")
+, systemdSupport ? stdenv.isLinux
+, systemd
```

```suggestion
, systemdSupport ? stdenv.isLinux, systemd
```

This is internally inconsistent styling with other single-package options.